### PR TITLE
{math}[intel/2017a] rjags v4-6, JAGS v4.2.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2573.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/236743df10957519c54a4728339ba7bf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2070.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fd1de3cde46da8141c8084fc93b3057e for a full test report.

File: easybuild/easyconfigs/j/JAGS/JAGS-4.2.0-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+homepage = 'http://mcmc-jags.sourceforge.net/'
+description = """JAGS is Just Another Gibbs Sampler.  It is a program for analysis 
+ of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  """
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [('http://sourceforge.net/projects/mcmc-jags/files/JAGS/%(version_major)s.x/Source/', 'download')]
+sources = [SOURCE_TAR_GZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/j/JAGS/JAGS-4.2.0-intel-2017a.eb]: Please add checksum

File: easybuild/easyconfigs/r/rjags/rjags-4-6-intel-2017a-R-3.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+description = """The rjags package is an interface to the JAGS library."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [
+    'http://cran.r-project.org/src/contrib/',
+    'http://cran.r-project.org/src/contrib/Archive/rjags/',
+]
+sources = ['%(name)s_%(version)s.tar.gz']
+
```
Comment (vanzod) [inline easybuild/easyconfigs/r/rjags/rjags-4-6-intel-2017a-R-3.4.0.eb]: Please add checksum

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2070.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5cd88525a5ae41cc880fd20b99fa6e33 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**